app/models/oclc/lc_call_slips/keyword_field.rb

```diff
@@ -41,7 +46,7 @@ def word_is_keyword?(word)
       end
 
       def normalize(word)
-        word.sub(/[[:punct:]]?$/, '')
+        normalize_keyword(word)[0]
```

Comment:
Instead of the original Ruby implementation, use the `normalize_keyword` function that we attached earlier.